### DISCUSSION TITLE
After the certifigets are generated do I need to keep port 80 open?

### DISCUSSION THREAD

**Discussion by prairietree** [nginx-proxy/acme-companion#1013] [Q&A] [answered]
The docker hub documentation says that both port 80 and 443 must be open for http-01 challenge. That makes sense. But if I am not planning on adding any more domains/sub-domains may I close port 80? Or is port 80 needed for renewal?

Why? Just a thing I have about closing all ports not absolutely required. 

Also, this is a really great container! It worked for me and was simple to set up.

Thank you. :)

**Comment by buchdag** [comment, marked as answer]
Hi, [keeping the port 80 opened is the official recommended practice from Let's Encrypt themselves](https://letsencrypt.org/docs/allow-port-80/). I entirely agree with them that closing port 80 for the sake of closing a port have zero security benefits and, in this case, only drawbacks.
